go/internal/controller/translator/labels/constants.go

```diff
@@ -0,0 +1,17 @@
+package labels
+
+// Standard Kubernetes label keys as defined in the Kubernetes documentation
+// https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/
+const (
+	AppName      = "app.kubernetes.io/name"
+	AppInstance  = "app.kubernetes.io/instance"
+	AppVersion   = "app.kubernetes.io/version"
+	AppComponent = "app.kubernetes.io/component"
+	AppPartOf    = "app.kubernetes.io/part-of"
+	AppManagedBy = "app.kubernetes.io/managed-by"
```

Comment:
We should proibably use these for agents as well, or at least decide on a shared set of labels. This can also be a TODO